src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import { getNumbers } from "../../utils";
+import cn from "classnames";
+
+type Props = {
+  currentPage: number;
+  items: string[];
+  perPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  currentPage,
+  items,
+  perPage,
+  onPageChange,
+}) => {
+  const handlePrevPage = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleNextPage = () => {
+    if (currentPage < Math.ceil(items.length / perPage)) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={cn('page-item', {
+          disabled: currentPage === 1,
+          })}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            onClick={handlePrevPage}
+            aria-disabled={currentPage === 1 ? 'true' : 'false'}
```

Comment:
The `href` attribute here is set to '#prev', which might alter the URL when clicked. If the intention is not to use the hash for navigation, consider using a button element instead of an anchor tag, or prevent the default action as mentioned in previous comments.